pvnet/models/multimodal/multimodal.py

```diff
@@ -56,105 +52,79 @@ def __init__(
         min_sat_delay_minutes: Optional[int] = 30,
         nwp_forecast_minutes: Optional[int] = None,
         nwp_history_minutes: Optional[int] = None,
-        sat_image_size_pixels: int = 64,
-        nwp_image_size_pixels: int = 64,
-        number_sat_channels: int = 12,
-        number_nwp_channels: int = 10,
-        source_dropout=0.0,
+        pv_history_minutes: Optional[int] = None,
         optimizer: AbstractOptimizer = pvnet.optimizers.Adam(),
     ):
-        """Neural network which combines information from different sources.
-
-        Args:
-            output_quantiles: A list of float (0.0, 1.0) quantiles to predict values for. If set to
-                None the output is a single value.
-            image_encoder: Pytorch Module class used to encode the NWP data (and satellite data
```

Comment:
Why would this be removed? I think having the options described can still be quite helpful

Reply:
Oops thanks, I meant to go back and update this